### ISSUE TITLE
Couldn't connect to accessibility bus: Failed to connect to socket

### ISSUE THREAD

**Issue by AlexWayfer** [ruby-gnome2/ruby-gnome2#1023]
Code:

```ruby
require 'gtk3'

window = Gtk::Window.new('First example')
window.set_size_request(400, 400)
window.set_border_width(10)

button = Gtk::Button.new(label: 'Say hello')
button.signal_connect 'clicked' do |_widget|
	puts 'Hello World!'
end

window.add(button)
window.signal_connect('delete-event') { |_widget| Gtk.main_quit }
window.show_all

Gtk.main
```

Output on start:

```
(null)-WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-slEpTCbz1J: Connection refused
	from /home/alex/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/gtk3-3.1.3/lib/gtk3/loader.rb:37:in `call_init_function'
	from /home/alex/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/gtk3-3.1.3/lib/gtk3/loader.rb:26:in `pre_load'
	from /home/alex/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/gobject-introspection-3.1.3/lib/gobject-introspection/loader.rb:38:in `load'
	from /home/alex/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/gtk3-3.1.3/lib/gtk3.rb:66:in `init'
	from /home/alex/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/gtk3-3.1.3/lib/gtk3.rb:40:in `const_missing'
	from gtk3.rb:3:in `<main>'
```

But GUI works fine.

Ruby: 2.3.4 and 2.4.1

Gtk3: 3.22.12

**Comment by kou** [comment]
It seems that it's your environment problem.
Can you run `gtk3-demo` provided by GTK+ 3 on your environment? And does it also print the warning?

**Comment by AlexWayfer** [comment]
@kou Yes, it does.

```
$ gtk3-demo

** (gtk3-demo:6145): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-aBO4fH5UVs: Connection refused
```

Thank you. I'm sorry.
